llvm/test/Feature/Repo/repo_pruning_dependents_1.ll

```diff
@@ -1,27 +1,41 @@
-; This testcase tests the program repository pruning pass.
+; This testcase tests the program repository pruning pass for the dependents.
+; The explanation of the dependents is given in the repo_pruning_dependents.ll,
+; Please refer to the file for details.
 ;
-; If functions 'A' and 'B' are both dependent on function 'C',
-; only add a single TicketNode of 'C' to the 'repo.tickets'
-; in order to avoid multiple compilation_members of function
-; 'C' in the compilation.
+; This test checks there is no redundant compilation members added into the compilation.
```

Comment:
I think this would be clearer:

This test checks **that** no redundant compilation members **are** added into the compilation.

Reply:
Fixed, thanks.